Explorer/Assets/DCL/PluginSystem/World/GltfContainerPlugin.cs

```diff
@@ -60,7 +60,13 @@ public void InjectToWorld(ref ArchSystemsWorldBuilder<Arch.Core.World> builder,
         {
             var buffer = sharedDependencies.EntityEventsBuilder.Rent<GltfContainerComponent>();
 
-            LoadGLTFSystem.InjectToWorld(ref builder, new NoCache<GLTFData, GetGLTFIntention>(false, false), sharedDependencies.SceneData, webRequestController);
+            LoadGLTFSystem.InjectToWorld(
+                ref builder,
+                new NoCache<GLTFData, GetGLTFIntention>(false, false),
```

Comment:
Don't create a new instance of `NoCache`, use `NoCache.INSTANCE` instead

Reply:
Tackled at https://github.com/decentraland/unity-explorer/pull/3097/commits/956c124a7ee0aa11682797b4ef21ab54d3d8bc90